examples/iowa.py

```diff
@@ -22,3 +22,10 @@
     'height': '4',
     'width': '7.5'
 }
+
+# Thumbnail Support
+THUMBNAIL = False
```

Comment:
This line should not be in here. It looks like there was a merge issue with this as well.